Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -10390,3 +10390,21 @@
     engine: WebKit
     engine_version: "536.25"
     family: ""
+-
+  user_agent: NetworkingExtension/8619.1.26.30.5 Network/4277.2.5 iOS/18.0
+  client:
+    type: browser
+    name: Mobile Safari
+    version: 8619.1.26.30
```

Comment:
wrong version for safari browser

The browser is supplied as part of the system and the browser version is always equal to the iOS version.
The browser is updated with the system update, there are no other ways.
Also, iOS cannot have any browser engines other than WebKit.